src/config/imp.rs

```diff
@@ -12,10 +12,12 @@ use std::path::Path;
 use thiserror::Error;
 use topological_sort::TopologicalSort;
 
+use super::ConfigIdent;
+
 /// Describes a set of packages to act upon.
 ///
 /// This structure maps "package name" to "package"
```

Comment:
Do you think it's worth it to make a thin wrapper around `ConfigIdent ` to label this as a `PackageName` specifically?

I'm assuming that with other configuration parameter usage, there will be other uses of `ConfigIdent` that are semantically different.

(If you want to punt this beyond this PR, I'm also fine with that)

Reply:
Per discussion in DMs, ended up creating three newtypes -- one each for package names, service names and preset names.